scripts/reformat.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
```

Comment:
could you add the `-euo pipefail` flag, just for reducing the amount of mistakes in the future, in case this scripts get more complex, also due to the usage of pipes

Reply:
Sure